test/contracts/managers/socialTradingManager.spec.ts

```diff
@@ -176,10 +178,14 @@ contract('SocialTradingManager', accounts => {
     );
     liquidatorWhiteList = await protocolHelper.deployWhiteListAsync([liquidator, newLiquidator]);
 
+    feeCalculator = await protocolHelper.deployFixedFeeCalculatorAsync();
+    feeCalculatorWhiteList = await protocolHelper.deployWhiteListAsync([feeCalculator]);
+
```

Comment:
You don't need to approve the feeCalculator to the whitelist?

Reply:
It is on deployment of the whitelist

Reply:
ohhh i see it. nice